mailbox/api/src/main/java/org/apache/james/mailbox/ApplicableFlagBuilder.java

```diff
@@ -24,42 +24,48 @@
 
 public class ApplicableFlagBuilder {
 
-    @VisibleForTesting
-    static final Flags DEFAULT_APPLICABLE_FLAGS = FlagsBuilder.builder().add(
-        Flags.Flag.ANSWERED,
-        Flags.Flag.DELETED,
-        Flags.Flag.DRAFT,
-        Flags.Flag.FLAGGED,
-        Flags.Flag.SEEN)
-        .build();
-
-    private final FlagsBuilder builder;
+    @VisibleForTesting static final Flags DEFAULT_APPLICABLE_FLAGS = FlagsBuilder.builder()
```

Comment:
* This means having the parameters at the same level of indent than the chained methods, not so good.
* And also, I really prefer to have only one chained method per line, such as in stream processing